.github/workflows/pull_request.yml

```diff
@@ -4,16 +4,14 @@ on:
   push:
     branches:
       - main
```

Comment:
Based on the docs [1], the workflow file is taken from the target branch, so we can even exclude main and leave just 7.x and 7.x.x branches. For main we could set it up similarly - main and every forked branch from it (8.x.x). Wdyt?

Reply:
I do not see [1] but looks logical to keep the branch checks specification within the concrete branch

Reply:
Thanks @MarianMacik @dupliaka ! I've just sent another commit which removes the main branch from the check. Will do the same on main branch.